Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace AreaAndVolume
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Print the length of a side: ");
+            int length = Convert.ToInt32(Console.ReadLine());
+
+            double volume = 5 * (3 + Math.Pow(5, 0.5)) * Math.Pow(length, 3) / 12;

```

Comment:
Квадратный корень удобнее вычислять с помощью метода `Math.Sqrt()`. Небольшие степени (квадрат, куб) лучше вычислять умножением. Так код читается легче.
```suggestion
            double volume = 5 * (3 + Math.Sqrt(5)) * length * length * length  / 12;

```